x-pack/plugins/elastic_assistant/server/routes/user_conversations/bulk_actions_route.ts

```diff
@@ -182,7 +182,7 @@ export const bulkActionConversationsRoute = (
               perPage: 100,
               page: 1,
               filter: `users:{ ${userFilter} } AND (${body.create
-                .map((c) => `title:${c.title}`)
+                .map((c) => `title:${JSON.stringify(c.title)}`)
```

Comment:
```suggestion
                // without stringify, special characters in the title can break this filter
                .map((c) => `title:${JSON.stringify(c.title)}`)
```
add a comment please!